README.md

```diff
@@ -387,12 +389,13 @@ The following settings apply to both the `JsonRpcServer` and `JsonServiceExporte
 Methods are resolved in the following way, each step immediately short circuits the
 process when the available methods is 1 or less.
 
-  1. All methods with the same name as the request method are considered
-  2. If `allowLessParams` is disabled methods with more parameters than the request are removed
-  3. If `allowExtraParams` is disabled then methods with less parameters than the request are removed
-  4. If either of the two parameters above are enabled then methods with the lowest difference in parameter count from the request are kept
-  5. Parameters types are compared to the request parameters and the method(s) with the highest number of matching parameters is kept
-  6. If there are multiple methods remaining then the first of them are used
+  1. If a method has the @JsonRpcMethod annotation, then if the annotation value has the same name as the request method, it is considered.  If the annotation has `required` set to `true`, then the Java method name is not considered.
+  1. Otherwise, all methods with the same name as the request method are considered.
```

Comment:
oh.. that is the markdown notation for a numbered list; it will render correctly.  Since I was inserting something at the top of the list, I had to renumber them anyway, and just set them to that.  Do you want it numbered explicitly?